uco-core/core.ttl

```diff
@@ -246,21 +231,24 @@ core:UcoObject
 			a owl:Restriction ;
 			owl:onProperty core:id ;
 			owl:cardinality "1"^^xsd:nonNegativeInteger ;
+			owl:onClass <http://unifiedcyberontology.org/identity#Identity> ;
```

Comment:
This is incorrect. core:id is not an Identity.
It is an Identifier. <http://unifiedcyberontology.org/types#Identifier>

Reply:
how in the world is this not causing errors to be thrown?

Reply:
Id has the Range of Identity... 
![image](https://user-images.githubusercontent.com/9854293/61142874-a6598100-a485-11e9-96ff-c325d2d683f9.png)

Reply:
![better-off-dead-tentacles-testicles-quote](https://user-images.githubusercontent.com/2760288/61143244-a8303e00-a49f-11e9-927c-2043ea937613.jpg)

:-)